src/screens/text-to-nft/generate-art/tabs/create/hooks/use-user-generation-quota.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+
+import { getStableDiffusionUserQuota } from 'src/apis/stable-diffusion';
+import { useAccessTokenSelector } from 'src/store/text-to-nft/text-to-nft-selectors';
+import { useSelectedAccountSelector } from 'src/store/wallet/wallet-selectors';
+import { isDefined } from 'src/utils/is-defined';
+
+export const useUserGenerationQuota = () => {
```

Comment:
Sorry, the quota works not in this way. A user is allowed to create 100 images in 24 hours window per one account. Storing the history of API calls makes no sense because a user may try using one account from two devices. That's why you should use `/api/v1/quota` API entry